src/bzflag/BackgroundRenderer.cxx

```diff
@@ -853,8 +847,8 @@ void BackgroundRenderer::drawSkybox()
     if (!BZDBCache::drawGround)
     {
         tm.bind(skyboxTexID[5]); // bottom
-        glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_WRAP_S, skyboxWrapMode);
-        glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_WRAP_T, skyboxWrapMode);
+        glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_WRAP_S, GL_CLAMP_TO_EDGE);
+        glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_WRAP_T, GL_CLAMP_TO_EDGE);
         glBegin(GL_TRIANGLE_STRIP);
```

Comment:
This is begging for a function (or at least a lambda) to reduce the amount of copy/paste

Reply:
It seems the pattern for introducing a 3d texture there